src/transmitter.ts

```diff
@@ -0,0 +1,29 @@
+import { Action } from './models/action';
+import { LIB_ID } from './models/constants';
+import { DEFAULT_OPTIONS, PostQuecastOptions } from './models/options';
+
+export class Transmitter {
+  private options: PostQuecastOptions;
+
+  constructor(options: Partial<PostQuecastOptions> = {}) {
+    this.options = {
+      ...DEFAULT_OPTIONS,
```

Comment:
Is it necessary to "build" `this.options`? What is the case when it needs other `options` than those passed as constructor argument?